T3/bin/buildMergedInputs.sh

```diff
@@ -37,7 +37,7 @@ cp -v ${WD}/local.cfg ${WD}/local_all.cfg
 cd $CMSSW_BASE/
 if [[ $doTar == 1 ]]; then
   PInfo -n "$myName" "Tarring up CMSSW..."
-  tar --exclude-vcs -chzf cmssw.tgz src python biglib bin lib objs test external # h = --dereference symlinks
+  tar --exclude-vcs -chzf cmssw.tgz src/Panda* python biglib bin lib objs test external # h = --dereference symlinks
```

Comment:
Why?

In the past, we've had external dependencies and shipping the entire source code is useful.

Reply:
this is just what i use locally, because I am messy. it got committed by mistake, will revert